src/lib/layers/LaneDetails.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  import type { LineString } from "geojson";
+  import type { Feature } from "../../types";
+  import { onMount } from "svelte";
+  import { gjScheme, routeInfo } from "../../stores";
+  import LanePolygons from "./lane_details/LanePolygons.svelte";
+  import LaneMarkings from "./lane_details/LaneMarkings.svelte";
+  import IntersectionPolygons from "./lane_details/IntersectionPolygons.svelte";
+  import IntersectionMarkings from "./lane_details/IntersectionMarkings.svelte";
+  import HelpIcon from "../common/HelpIcon.svelte";
+
+  // Show along a route
+  export let id: number;
+
+  // renderLaneDetailsForRoute returns 4 layers in a certain order. They're
+  // labeled when used below.
+  let gj1;
+  let gj2;
+  let gj3;
+  let gj4;
+
+  onMount(async () => {
+    try {
+      let linestring = $gjScheme.features.find(
+        (f) => f.id == id
+      ) as Feature<LineString>;
+      let raw = await $routeInfo.renderLaneDetailsForRoute(
```

Comment:
Note for me. 

This is where the magic happens and we call the new function for rendering lane details from the webWorker